tests/test_proxy_functional.py

```diff
@@ -139,6 +147,10 @@ async def test_secure_https_proxy_absolute_path(
     await conn.close()
 
 
+@pytest.mark.xfail(
+    PY_310 and platform.system() != "Darwin",
+    reason="we need to fix the secure proxy fixture in 3.10",
```

Comment:
Let's be more precise here:
```suggestion
    raises=ConnectionAbortedError,
    reason="we need to fix the secure proxy fixture in 3.10",
```

Reply:
Yes, but I think it would be better if we make the decrator a partial function, only reuse the condition and reason arguments. Because the exception behavior is dependent on the logic of test function. This will increase coupling.